### DISCUSSION TITLE
Package not in package.json

### DISCUSSION THREAD

**Discussion by LiamJBerridge** [artilleryio/artillery#2254] [Artillery]
Hi,

I have created some playwright tests and I'm trying to run them using artillery on fargate and i'm getting the below error when budling the tests before execution. Tests that use these dependancies are failing, how do we bundle faker.js into the tests that are executed on fargate?
![Screenshot 2023-10-26 at 18 11 30](https://github.com/artilleryio/artillery/assets/45438533/769e6054-f268-44bb-af81-50dfe15631d9)

**Comment by bernardobridge** [comment]
Hi @LiamJBerridge, the `not in package.json` part is not an error, it means that those packages aren't bundled by default (so they're usually custom requires you're making), so it needs to install them in the worker.

However, you do have an error coming from the worker. You should be able to check the actual error by going into Cloudwatch in the AWS console (in the right region), and looking for logs under the log group `artilleryio-log-group/CLUSTER_NAME`. Then you can search by the right log based on the test ID (provided in the beginning of the test). _Note: you may need to run your test again_

Once you have the error that is happening in the worker, could you let us know what it is? We would be able to help troubleshoot then.

**Reply by LiamJBerridge** [reply]
Hi @bernardobridge, 

thanks for the help! 

It looks like faker is installing based on these 2 logs - 

2023-10-26T21:02:13.055+01:00 | installing @faker-js/faker
2023-10-26T21:02:15.696+01:00 | added 1 package, and audited 9 packages in 2s


It looks like the issue is with chromium not being on the fargate instance?

2023-10-26T21:02:33.120+01:00Copyworker error, id: 1 browserType.launch: Executable doesn't exist at /ms-playwright/chromium-1076/chrome-linux/chrome | worker error, id: 1 browserType.launch: Executable doesn't exist at /ms-playwright/chromium-1076/chrome-linux/chrome

**Reply by bernardobridge** [reply]
Hi @LiamJBerridge - which version of Artillery are you using? (you can check this with `artillery --version`)

**Reply by LiamJBerridge** [reply]
@bernardobridge 

VERSION INFO:

Artillery: 2.0.0-37
Node.js:   v18.18.0
OS:        darwin

**Reply by bernardobridge** [reply]
@LiamJBerridge can you try upgrading  to `artillery@2.0.0-38`? 

I believe the problem is that it's installing the `artillery-engine-playwright`, which is now already bundled, and that creates a browser version mismatch on the container. We solved this problem on `Artillery v2.0.0-38`, so it should work.

**Reply by LiamJBerridge** [reply]
@bernardobridge 

updating to v2.0.0-38 seems to have resolved the issue and it looks like the tests are running on fargate! Thanks for your help!
